youslist.txt

```diff
@@ -7503,6 +7505,8 @@ namu.wiki##:xpath(//div/h1[not(./small)]/following-sibling::div[1]/following-sib
 @@||cdn.kbmaeil.com/bannerpop/uploads/js/*.js^$domain=kbmaeil.com
 @@||googleads.g.doubleclick.net/pagead/ads?*&crui=*
 @@||mindpost.or.kr/script/bannerpop.js^$domain=mindpost.or.kr
+@@||script.gmarket.co.kr/js/common/pds_gmkt.js
+@@||script.gmarket.co.kr/js/header/statsna.js
```

Comment:
Will this be enough, or other sites also use these scripts?

```suggestion
@@||script.gmarket.co.kr/js/common/pds_gmkt.js^$domain=gmarket.co.kr
@@||script.gmarket.co.kr/js/header/statsna.js^$domain=gmarket.co.kr
```

Reply:
If that covers all subdomain then that looks good to me.